external.bzl

```diff
@@ -121,6 +121,13 @@ def go_dependencies():
         sum = "h1:BSKMNlYxDvnunlTymqtgONjNnaRV1sTpcovwwjF22jk=",
         version = "v1.0.10",
     )
+    go_repository(
+        name = "com_github_creack_pty",
+        importpath = "github.com/creack/pty",
+        sum = "h1:uDmaGzcdjhF4i/plgjmEsriH11Y0o7RKapEf/LDaM3w=",
+        version = "v1.1.9",
+    )
+
```

Comment:
nit: Can you manually remove those blank lines? It's kinda a bug of gazelle, that when it create this file for the first time there's no blank lines, but when it add some new dependencies it will add blank lines after those new dependencies, which is inconsistent.